src/ngraph/runtime/backend.cpp

```diff
@@ -29,6 +29,12 @@ runtime::Backend::~Backend()
 {
 }
 
```

Comment:
You should call it `get_backend_op` since it is returning a node that is an op, as opposed to a node that is some other thing like a pattern.

Reply:
Yes - that makes sense. However, this bubbles up an interesting question - the return type should be ngraph::Op and `not` ngraph::Node - is there such a thing?

Reply:
@avijit-nervana That does exist, yes.

Reply:
@aprocter That's what I thought - so in this case the return type should be changed to ngraph::Op to make it fail safe.

Reply:
`Op` is a child class of `Node`. At the bridge we mostly use `Op`s
Yes, we can change the return type to `Op`